test/apify_helpers.js

```diff
@@ -27,4 +31,32 @@ describe('apify utils', () => {
         };
         expect(prefillValues).to.be.eql(expected);
     });
+
+    describe('createFieldsFromInputSchemaV1() works', () => {
+        const validator = makeValidator(FieldSchema);
+
+        it('for Web Scraper', () => {
```

Comment:
Maybe we could also check that the number of fields returned is the same as number of properties in input schema?

Reply:
I'm checking if all fields from inputSchema.properties are in fields. It should be enough as I'm adding some extra fields (copy) so the count will not match.